### PR TITLE
builddir environment

### PR DESCRIPTION
**Pull request by felix-salfelder**
this fixes running qucs from builddir. this is required for the "check" target to work. (#376 rebased)

### CODE REVIEW

File: qucs/qucs/main.cpp [hunk 1]
```diff
@@ -706,7 +706,7 @@ int main(int argc, char *argv[])
   QucsSettings.Editor = "qucs";
 
   /// \todo Make the setting up of all executables below more consistent
-  var = getenv("QUCSATOR");
+  getenv("QUCSATOR");
```
Comment (guitorri) [inline qucs/qucs/main.cpp]: Isn't `var = getenv()` needed here?

File: qucs/qucs/Makefile.am [hunk 2]
```diff
@@ -42,7 +52,7 @@ MOCHEADERS = \
 
 MOCFILES = $(MOCHEADERS:.h=.moc.cpp)
 
-qucs_SOURCES = node.cpp element.cpp qucsdoc.cpp wire.cpp mouseactions.cpp   \
+qucs_real_SOURCES = node.cpp element.cpp qucsdoc.cpp wire.cpp mouseactions.cpp   \
```
Comment (guitorri) [inline qucs/qucs/Makefile.am]: Please enlighten me, why _real_ is needed here?

File: qucs/qucs/main.cpp (hunk 1 above)
Comment (felix-salfelder) [inline qucs/qucs/main.cpp]: !!

thanks

File: qucs/qucs/Makefile.am (hunk 2 above)
Comment (felix-salfelder) [inline qucs/qucs/Makefile.am]: "qucs", the executable is a wrapper script managed by autotools.

this PR introduces a custom wrapper script "qucs", generated from "qucs.in", it only lives in the builddir. here it executes "qucs_real".

the "_real" suffix for the autotools wrapper is arbitrary, it is stripped in line 43 (during install).

**Comment by felix-salfelder** [comment]
updated. still needed for testing in upcoming PRs...

**Comment by felix-salfelder** [comment]
can we have this in .20.? seems like a user (leon-vv) has aked for it in the meantime

**Comment by guitorri** [comment]
Testing.

**Comment by in3otd** [comment]
not sure it's related to this, but when I start `qucs` from the builddir I get a message on the console saying
`make: *** No rule to make target 'subdirs'.  Stop.`
but the GUI actually comes up and seems to work.
Any idea where that message comes from?

**Comment by felix-salfelder** [comment]
thanks for spotting this.

yes, there once was a `subdirs` target, which is used to compile `PATH` in the wrapper script.

    subdirs:
    \techo ${SUBDIRS}

(i don't know where it went, probably lost during a rebase)

**Pull request merged**